chef/cookbooks/bind9/recipes/default.rb

```diff
@@ -183,13 +183,11 @@ def make_zone(zone)
 # As DHCP addresses can be re-used, we make sure to use the one node which is
 # the most recent; this requires two passes
 temporary_dhcp = {}
-# Get the config environment filter
-#env_filter = "dns_config_environment:#{node[:dns][:config][:environment]}"
-env_filter = "*:*" # Get all nodes for now.  This is a hack around a timing issue in ganglia.
 # Get the list of nodes
-nodes = search(:node, "#{env_filter}")
+nodes = node_search_with_cache("*:*")
```

Comment:
No, it will just mean we don't use an additional filter string, and so we really only search for `"*:*"`.